classes/Language.php

```diff
@@ -1189,15 +1189,28 @@ public static function countActiveLanguages($id_shop = null)
         return static::$countActiveLanguages[$id_shop];
     }
 
-    public static function downloadAndInstallLanguagePack($iso, $version = _PS_VERSION_, $params = null, $install = true)
+    /**
+     * @param mixed $version Unused parameter @deprecated since 8.0.0 and will be removed in next major version.
+     */
```

Comment:
```suggestion
    /**
     * Downloads, then installs/updates a language pack
     *
     * @param string $iso 2-letter language code
     * @param mixed $version [deprecated since 8.0.0 - has no effect]
     * @param array $params Optional parameters (@see self::checkAndAddLanguage()) - only used if $install is true
     * @param bool $install [default=true] If false, update an existing language pack
     */
```